nemo/collections/nlp/modules/common/megatron/parallel_adapters.py

```diff
@@ -44,6 +44,8 @@ def __init__(
         dim: int,
         activation: str = 'swish',
         norm_position: str = 'post',
+        norm_type: str = 'mixedfused',
+        column_init: str = 'xavier',
```

Comment:
Is this column init because it only initialized the columnlinear layer within the adapter with this initialization scheme?

Reply:
We have initialization methods and kwargs for layers in `transformer.py` already, would it be possible to repurpose those?

Reply:
Also, can we rename it `init_method` rather than just `init`

Reply:
Renamed the options.